float32 accx
float32 accy
float32 accz
float32 gyrox
float32 gyroy
float32 gyroz
int32 hr
int32 velocity